backend/src/server.ts

```diff
@@ -63,6 +78,37 @@ export const context = async (options) => {
   }
 }
 
+export const loggerPlugin = {
```

Comment:
Maybe put this into its own file so we can test just the logger plugin with a test server?